components/supervisor/pkg/supervisor/supervisor.go

```diff
@@ -978,6 +978,7 @@ func buildChildProcEnv(cfg *Config, envvars []string) []string {
 	//
 	envs["HOME"] = "/home/gitpod"
 	envs["USER"] = "gitpod"
+	envs["NODE_OPTIONS"] = ""
```

Comment:
Should not we change code helper instead?

Reply:
Altought this works, I was also wondering where's the best place to do this. I will check in code-helper. 🙏

Reply:
If in supervisor we need to consider other clients, ie for task terminals it makes sense generally. 

There maybe is more generic solution to ignore user variables for IDE startup, not sure.

Reply:
Changed!